code/mob.dm

```diff
@@ -78,6 +78,8 @@
 	var/temp_tolerance = 15 // iterations between each temperature state
 	var/thermoregulation_mult = 0.025 // how quickly the body's temperature tries to correct itself, higher = faster
 	var/innate_temp_resistance = 0.16  // how good the body is at resisting environmental temperature, lower = more resistant
+	var/min_temp_mult = 0.1 // How much should temperature decrease the life mult, if applicable
+	var/max_temp_mult = 5 // How much should temperature increase the life mult, if applicable
```

Comment:
this seems a very extreme range

Reply:
Its a debug range, mostly to see if it actually worked. And it does!

Though, I do want a pretty big range. Maybe less than this!